config/yamlcfg/yaml2proto_test.go

```diff
@@ -254,6 +254,45 @@ func Test_MarshalYAML(t *testing.T) {
   name: dashboard_1
 test_groups:
 - name: testgroup_1
+`),
+		},
+		{
+			name: "column headers work poorly TODO(fejta): fix",
```

Comment:
Can you expand a bit here, perhaps in a comment? What is the TODO here?

Reply:
PR forthcoming